steps/set-ios-info-plist-unified/1.1.1/step.yml

```diff
@@ -0,0 +1,73 @@
+title: Set iOS Info.plist Values (unified)
+summary: Sets various values in the target Info.plist file for the next build.
+description: Sets various values in the target Info.plist file for the next build.
+website: https://github.com/mapletonhillmedia/steps-set-ios-info-plist
+source_code_url: https://github.com/mapletonhillmedia/steps-set-ios-info-plist
+support_url: https://github.com/mapletonhillmedia/steps-set-ios-info-plist/issues
+published_at: 2016-12-29T14:37:43.172584977-07:00
+source:
+  git: https://github.com/mapletonhillmedia/steps-set-ios-info-plist.git
+  commit: c727ccd56c882012d6d7013a469ab554ed82d971
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
```

Comment:
can you please add the `macos` project type too? it should work for macOS app projects as well